__init__.py

```
@@ -1,18 +1,23 @@
 import bpy
-from bpy.props import (PointerProperty)
+from bpy.props import PointerProperty
 from .four_d_humans_blender import *
```

Ruff

`from .four_d_humans_blender import *` used; unable to detect undefined names

(F403)